nbgrader/server_extensions/formgrader/templates/base.tpl

```diff
@@ -31,14 +31,19 @@
           <h1>nbgrader</h1>
         </div>
       </div>
-      <div class="col-md-10">
+      <div class="col-md-8">
         <div class="page-header">
           <h1>
           {%- block title -%}
           {%- endblock -%}
           </h1>
         </div>
       </div>
+      <div class="col-md-2">
+        <div class="jupyter-logo">
```

Comment:
I think it would look nice if the logo were right-aligned, which you can get by adding the `pull-right` class.